usr/lib/pkcs11/ep11_stdll/ep11_specific.c

```diff
@@ -7692,3 +7706,188 @@ CK_BBOOL ep11tok_optimize_single_ops(STDLL_TokData_t *tokdata)
 
     return ep11_data->optimize_single_ops ? CK_TRUE : CK_FALSE;
 }
+
+/* return -1 if v1 < v2, = iof v1 == v2, 1 if v1 > v2 */
```

Comment:
did you mean "0 if v1 == v2" ?

Reply:
Oups, yes of course. I have corrected it.